katportalclient/client.py

```diff
@@ -967,6 +999,7 @@ def schedule_blocks_assigned(self):
         SubarrayNumberUnknown:
             - If a subarray number could not be determined.
         """
+        yield self._init_sitemap()
```

Comment:
As discussed @bmerry, would be nice if the init and reading of a field from the sitemap could be combined into a single coroutine, dropping the property access.